### PR TITLE
transition for accepting back office assets for loan

### PR DESCRIPTION
**Pull request by MarekBleschke**

### CODE REVIEW

File: src/ralph/admin/templatetags/dashboard_tags.py [hunk 1]
```diff
@@ -47,6 +47,20 @@ def get_user_equipment_to_accept_tile_data(user):
     }
 
 
+def get_user_equipment_to_accept_loan_tile_data(user):
+    from ralph.accounts.helpers import \
```
Comment (mkurek) [inline src/ralph/admin/templatetags/dashboard_tags.py]: why do you import it in function, not at the top of the module?

File: src/ralph/accounts/helpers.py [hunk 2]
```diff
@@ -9,20 +10,38 @@
...
-        id=ACCEPTANCE_TRANSITION_ID
+        id=transition_id
     ).exists()
 
 
-def get_assets_to_accept(user):
+acceptance_transition_exists = partial(
+    transition_exists, ACCEPTANCE_TRANSITION_ID
+)
+loan_transition_exists = partial(
```
Comment (mkurek) [inline src/ralph/accounts/helpers.py:26]: is it used anywhere?

File: src/ralph/admin/templatetags/dashboard_tags.py (hunk 1 above)
Comment (MarekBleschke) [reply src/ralph/admin/templatetags/dashboard_tags.py]: I was basing this on code for accepting equipment. Actually it seems to work with importing this  on module level so I've moved it there.

File: src/ralph/accounts/helpers.py (hunk 2 above)
Comment (MarekBleschke) [reply src/ralph/accounts/helpers.py:26]: Good catch :)
I've missed part in accepting equipment on "my equipment" page. Added changes there for loan so this function is now used there.

**Comment by mkurek** [comment]
ok for me, but travis has some complaints :)

**Comment by MarekBleschke** [comment]
@mkurek fixed tests, travis is happy again :)

**Review by mkurek — APPROVED** [review]

**Review by ar4s — APPROVED** [review]

**Pull request merged**